src/Abp/WebHooks/WebHookInfo.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations.Schema;
+using Abp.Domain.Entities;
+using Abp.Domain.Entities.Auditing;
+
+namespace Abp.WebHooks
+{
+    /// <summary>
+    /// Store created web hooks. To see who get that webhook check with <see cref="WebHookWorkItem.WebHookId"/> and you can get <see cref="WebHookWorkItem.WebHookSubscriptionId"/>
+    /// </summary>
+    [Table("AbpWebHooks")]
+    public class WebHookInfo : Entity<Guid>, IMayHaveTenant, IHasCreationTime, ISoftDelete
```

Comment:
i think if we implements `ISoftDelete` here, we probably should implements  `IHasDeletionTime`